coderedcms/templates/coderedcms/blocks/hero_block.html

```diff
@@ -4,7 +4,7 @@
 <div class="container">
 {% endif %}
 
-{% image self.background_image max-2000x2000 as background_image %}
+{% image self.background_image original as background_image %}
```

Comment:
So in this case, `max-2000x2000` is not changing the proportions, but ensuring that the image is not greater than 2,000px in either direction (e.g. if someone uploads a giant 10MB 50,000px  50,000px image).

I'm not sure if we want to remove that safeguard?

Reply:
Okay, reverting it back.